resolwe_api/resolwe.py

```diff
@@ -189,16 +213,18 @@ def processes(self, process_name=None):
         :rtype: list of Process objects
 
         """
+        # TODO: Make a better representation? If this is run from command line
+        # it outputs thousands of lines... Not really helpful.
         if process_name:
             return [p for p in self.api.process.get() if process_name in p['name']]
         else:
             return self.api.process.get()
 
-    def printad_processes(self):
+    def print_upload_processes(self):
         """Print all upload process names."""
         for p in self.processes():
             if 'upload' in p['category']:
-                print(p['name'])
+                sys.stdout.write(p['name'] + '\n')
```

Comment:
Maybe change it to [Python's logger](https://docs.python.org/2/library/logging.html). We will need to log the output when it runs as a daemon on the server. It would be nice if you could tell it with some setting where to log. The default could be stdout. @dblenkus, what do you think about it?